types/api/public-offer-v2.d.ts

```diff
@@ -551,6 +506,70 @@ export namespace PublicOfferV2 {
     type: LeTourOfferType;
   }
 
+  interface LeOfferSkeleton {
+    id: string;
+    name: string;
+    slug?: string;
+  }
```

Comment:
offer skeleton should have a list of options and these are the things someone can buy
```suggestion
interface OptionBase {
   price: number;
   id: string;
}

  interface LeOfferSkeleton {
    id: string;
    name: string;
    slug?: string;
    options: Array<OptionBase>;
  }
```

Reply:
Yep I haven't included prices as not sure if its required in the new tour offer page. I will add it in now to show you what I'm thinking.